app/screens/login/login.test.js

```diff
@@ -114,4 +116,41 @@ describe('Login', () => {
 
         // This test times out if resetToChannel hasn't been called
     });
+
+    test('should go to MFA screen when login response returns MFA error', (done) => {
+        baseProps.actions.goToScreen.mockImplementation(() => {
+            done();
```

Comment:
Does this actually hit the expect statement? I feel like `done` is going to be called before we check the arguments